packages/perseus/src/widgets/interactive-graph.tsx

```diff
@@ -965,11 +965,7 @@ class LegacyInteractiveGraph extends React.Component<Props, State> {
             throw makeInvalidTypeError("createPointForPolygonType", "polygon");
         }
 
-        // TODO(alex): check against "grid" instead, use constants
-        const snapToGrid = !_.contains(
-            ["angles", "sides"],
-            this.props.graph.snapTo,
-        );
+        const snapToGrid = this.props.graph.snapTo === "grid";
```

Comment:
Completing the TODO from Alex, though I tried to match how other parts of the code does this.